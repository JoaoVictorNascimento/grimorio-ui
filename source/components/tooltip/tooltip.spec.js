import '../../../internals/test/helper';

import Tooltip from './tooltip-component';

/** @test {Tooltip} */
describe('Tooltip component', () => {
/** @test {Tooltip#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Tooltip message="hello" />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
