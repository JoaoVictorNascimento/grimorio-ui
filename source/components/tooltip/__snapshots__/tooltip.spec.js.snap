// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip component should render with default props 1`] = `
"<span className=\\"wrapper\\">
  <Icon name=\\"info\\" size={19} align=\\"middle\\" />
  <span style={{...}} className=\\"dialog top\\">
    hello
  </span>
</span>"
`;
